src/locales/en-US/list.ftl

```diff
@@ -2,10 +2,31 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-## common
+## Firefox / Lockbox Brand
+##
+## Firefox and Lockbox must be treated as a brand, and kept in English.
+## They cannot be:
+## - Declined to adapt to grammatical case.
+## - Transliterated.
+## - Translated.
+##
+## Reference: https://www.mozilla.org/styleguide/communications/translation/
+-brand-name = Firefox
+-product-short-name = Lockbox
+-product-full-name = Firefox Lockbox
+# TODO: should Firefox Account also not be translated or declined?
```

Comment:
No, Firefox account is a different beast.
https://hg.mozilla.org/l10n/gecko-strings/file/default/browser/browser/branding/sync-brand.ftl#l11

It would be great to use the same message ID for it, in case there are matches in translation memory.